RetroBar/Languages/English.xaml

```diff
@@ -57,6 +57,13 @@
         <s:String>Open a new instance</s:String>
         <s:String>Close the task</s:String>
     </x:Array>
+    <s:String x:Key="clock_click_action">_Clock click action:</s:String>
+    <x:Array x:Key="clock_click_action_values" Type="s:String">
+        <s:String>Do nothing</s:String>
+        <s:String>Open Aero clock flyout</s:String>
+        <s:String>Open Action Center/Notification Center</s:String>
+        <s:String>Open Modern calendar</s:String>
```

Comment:
What do you think about using the following:

- Open Aero calendar
- Open Modern calendar
- Open Notification Center

Reply:
I guess those are more familiar, I wanted to use the ones StartAllBack uses, but yeah, I'll change them!
![SABUsesFlyout](https://github.com/user-attachments/assets/1401e60b-37e9-49bb-836b-56ca12b346df)